tools/logger/convert.c

```diff
@@ -681,22 +681,11 @@ static int verify_fw_ver(const struct convert_config *config,
 		return -EINVAL;
 	}
 
-	/* logger and version_file abi dbg verification */
-	if (SOF_ABI_VERSION_INCOMPATIBLE(SOF_ABI_DBG_VERSION,
-					 ver.abi_version)) {
-		log_err(config->out_fd,
-			"abi version in %s file does not coincide with abi version used by logger.\n",
-			config->version_file);
-		log_err(config->out_fd,
-			"logger ABI Version is %d:%d:%d\n",
-			SOF_ABI_VERSION_MAJOR(SOF_ABI_DBG_VERSION),
-			SOF_ABI_VERSION_MINOR(SOF_ABI_DBG_VERSION),
-			SOF_ABI_VERSION_PATCH(SOF_ABI_DBG_VERSION));
+	/* compare firmware hash value from version file and ldc file */
+	if (ver.fw_hash != snd->version.fw_hash) {
```

Comment:
doesn't `memcmp()` above already compare the two versions?

Reply:
True, this `memcmp()` also needs to be removed, because of intended ABI/DBG_ABI mismatch. Updated